driver-core/src/test/resources/unified-test-format/gridfs/delete.json

```diff
@@ -1,301 +1,280 @@
 {
-  "description": "gridfs-delete",
-  "schemaVersion": "1.0",
-  "createEntities": [
+  "description" : "gridfs-delete",
```

Comment:
The commit with the changes that were supposed to be merged to our repository, removed 60 lines from `delete.json`, yet here we many more lines affected. The issue seem to be caused by the extra spaces before `:`, which are not supposed to be there, and was likely introduced by auto-formatting you have enabled in the IDE.

The rest of the modified files have the same issue.

Reply:
Sorry, I was unaware about the auto-formatting. Will push in the changes with auto-formatting disabled!